lib/resources/dstu2/soarian_device.yaml

```diff
@@ -9,13 +9,13 @@ fields:
   description: Identifier(s) from manufacturer, owner and others.
   children:
 
-  
+
```

Comment:
can just take this line out rather than remove spaces from it

Reply:
done: https://github.com/cerner/fhir.cerner.com/pull/250/commits/1c50d0d44d939152be7cb5a99f303209e8c89175#diff-fd3715a297194d5a88382f188ee54252L11